tools/cfn2ts/lib/codegen.ts

```diff
@@ -581,7 +588,7 @@ export default class CodeGenerator {
         if (union.indexOf('|') !== -1) {
           alternatives.push(`Array<${union}>`);
         } else {
-          alternatives.push(`(${union})[]`);
+          alternatives.push(`${union}[]`);
```

Comment:
was this dead code? It broke for me and changing this fixes it, but it does leave me asking how did this work if it wasn't dead code?